### ISSUE TITLE
Voice/Video call between members in a relation

### ISSUE THREAD

**Issue by rudrakshjhalanii07** [anitab-org/mentorship-backend#413]
**Is your feature request related to a problem? Please describe.**
What I felt is improving one's career could be a cumbersome task if the only media to communicate is through chats. It would be a great option if the members in a current relation could communicate with each other over voice/video calls 

**Describe the solution you'd like**
A voice and video call feature available for both the members in the relation.
Here, two conditions would be mandatory for making a call,
Firstly, two members must be in a current mentorship relation to get the call feature available.
Secondly, both the members must have given prior permission for making calls available, only then a call would be possible between the two members.

**Comment by bartekpacia** [comment]
This is a good idea, but I feel that first, we should make text communication working and then think about something more advanced. What do you think?

**Comment by rudrakshjhalanii07** [comment]
Well of course @bartekpacia , infact this feature would be incorporated in the chat page only, without that it would be just a mess😅

**Comment by bartekpacia** [comment]
Yeah, that's right :D

**Comment by vj-codes** [comment]
@isabelcosta closing since this feature won't be implemented soon and is off theme of mentorship-system.

**Comment by SanketDG** [comment]
Can we put a `on-hold` or `Future` label maybe?

**Comment by PrashanthPuneriya** [comment]
Yes, it's better to not close such type of issues. Otherwise they will be buried down the closed issues hole :D. To my knowledge these type of issues comes under `Outreach/Research` category.
